packages/smithy-client/src/client.ts

```diff
@@ -1,11 +1,19 @@
 import { constructStack } from "@smithy/middleware-stack";
-import { Client as IClient, Command, MetadataBearer, MiddlewareStack, RequestHandler } from "@smithy/types";
+import {
+  Client as IClient,
+  Command,
+  FetchHttpHandlerOptions,
+  MetadataBearer,
+  MiddlewareStack,
+  NodeHttpHandlerOptions,
+  RequestHandler,
+} from "@smithy/types";
 
 /**
  * @public
  */
 export interface SmithyConfiguration<HandlerOptions> {
```

Comment:
This is an input config corresponding to the resolved config below. So, it can be loosened.